README.md

```diff
@@ -29,6 +32,17 @@ sudo apt-get install libwebkit2gtk-4.0-dev
 Open the root `CMakeLists.txt` in CLion. Then the project can be built with 
 `Ctrl+F9` and run with `Shift+F10`.
 
+# MacOS
+
+## Requirements
+
+* CMake
+* WebKit framework (should come preinstalled)
+
+## Building and running
+
+Todo: instructions
```

Comment:
@fdidron Would you mind adding how you build the project on MacOS here?

Reply:
@wilricknl Not sure how one would import the project into Xcode as I don't use it.

What I did from the projects root:
```
mkdir build
cd build
cmake ..
make
```